### PR TITLE
Minidump: Disable TF 105000 in the to avoid ConstantEvaluation [#116803217]

### PR DESCRIPTION
**Pull request by vraghavan78**
In the pull request, I have disabled the constant expression evaluation when there is none provided in the particular unit test. 

There is two tests where the predicate seems to have swapped sides. Not sure if these are non-deterministic tests.

@xinzweb @d @oarap @hsyuan please take a look.

### CODE REVIEW

**Comment by hsyuan** [comment]
LGTM

**Comment by oarap** [comment]
+1

File: libgpopt/src/minidump/CMinidumperUtils.cpp [hunk 1]
```diff
@@ -418,6 +418,16 @@ CMinidumperUtils::PdxlnExecuteMinidump
 	CBitSet *pbsDisabled = NULL;
 	SetTraceflags(pmp, pdxlmd->Pbs(), &pbsEnabled, &pbsDisabled);
 
+	BOOL fConstantExpressionEvaluator = true;
```
Comment (xinzweb) [inline libgpopt/src/minidump/CMinidumperUtils.cpp]: Seems like this code got duplicated.

Maybe we can take this chance to refactor this function by reusing the second function with the new MDAccessor?

Comment (vraghavan78) [inline libgpopt/src/minidump/CMinidumperUtils.cpp]: its 3 lines, and I did not want add more functions for that :(

**Pull request closed without merging**